ci/azure-functions-dotnet-windows.yml

```diff
@@ -0,0 +1,43 @@
+name: Build and deploy .NET Core Function App to Azure functions app
+
+on:
+  push:
+    branches: [ $default-branch ]
+  
+  workflow_dispatch:
+
+env:
+  AZURE_FUNCTIONAPP_NAME: 'YOUR_FUNCTIONAPP_NAME'
+  AZURE_FUNCTIONAPP_SLOT_NAME: 'Production' # Change it according to your project
+  AZURE_FUNCTIONAPP_PACKAGE_PATH: './publish' # Change it according to your project
+  CONFIGURATION: Release
+  DOTNET_CORE_VERSION: '3.1' # Change it according to your project
+
+jobs:
+  build-and-deploy:
+    runs-on: windows-latest
```

Comment:
The only difference with linux workflow is the runner?
If so, can't we use ubuntu runner for both, and hence having one single template?

Reply:
Yes, that is the only difference. Steps followed in the windows based function app deployment can be followed in Linux based, and it will still work.

Reply:
But in the long run, we would want to showcase the affinity of tech stacks w.r.t. OS of the deployment target, thus from that perspective, having two separate templates based on OS gives us an extra edge in that direction.

Reply:
Although I agree this is just redundant in current scenario. Open to merge these two and have a single template for now. Thoughts? @bishal-pdMSFT @ashwinsangem ?